episodes/00-before-we-start.Rmd

```diff
@@ -282,6 +282,11 @@ selected text will be sent to the console and executed when you press
 this [RStudio cheatsheet about the RStudio
 IDE](https://raw.githubusercontent.com/rstudio/cheatsheets/main/rstudio-ide.pdf).
 
+To run an entire script you can use the <kbd>`Ctrl`</kbd> +  <kbd>`Shift`</kbd> + 
+<kbd>`Enter`</kbd> shortcut (on Macs, <kbd>`Cmd`</kbd> + <kbd>`Shift`</kbd> + 
+<kbd>`Return`</kbd>) or use the "Source" toolbar button on the 
```

Comment:
```suggestion
To run an entire script you can use the <kbd>Ctrl</kbd> +  <kbd>Shift</kbd> + 
<kbd>Enter</kbd> shortcut (on Macs, <kbd>Cmd</kbd> + <kbd>Shift</kbd> + 
<kbd>Return</kbd>) or use the "Source" toolbar button on the 
```

just a suggestion: remove the code tags because it's wrapped in `<kbd>` tags, which already formats it nicely.

Reply:
thanks! I used the backticks to stay consistent with the rest of the rmd. The spacing appears to change slightly if the backticks are omitted 

![image](https://github.com/datacarpentry/R-ecology-lesson/assets/23587234/33a952f3-36e8-42e9-bb46-6117e76323ca)